layers/+frameworks/emberjs/packages.el

```diff
@@ -0,0 +1,75 @@
+;;; packages.el --- emberjs layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Robert O'Connor <robby.oconnor@gmail.com>
+;; URL: https://github.com/robbyoconnor
```

Comment:
I'm not sure if this is right...

Reply:
I think the url should reflect the canonical location from where you can get the code, aka the spacemacs repo.

Reply:
I fixed it -- wasn't 100% sure